index.html

```diff
@@ -0,0 +1,43 @@
+                <a href="projects.html" class = "links" style = "padding-left: 15px; padding-right: 15px; text-decoration:none">PROJECTS</a>
+                <a href="team.html" class = "links"style = "padding-left: 15px; padding-right: 15px; text-decoration:none">TEAM</a>
+                <a href="https://secure.ucsc.edu/s/1069/bp18/interior.aspx?sid=1069&gid=1001&pgid=780&cid=1749&dids=1207" class="links" style="padding-left: 15px; padding-right: 15px; text-decoration:none" target="_blank">DONATE</a>
+            </div>
+        </div>
+        <img src = "_assets/giving_day_banner2.png" style = "max-width:100%; height:auto;">
+        <div style = "font-family: 'Orbitron', sans-serif;font-size:20px; color:white; padding-top: 80px; text-align: center; font-size: 50px;"> WELCOME TO AUTOSLUG</div>
+        <div style = "font-family: 'Orbitron', sans-serif;font-size:20px; color:white; padding-top: 80px; text-align: center; padding-left: 20px; padding-right: 20px; padding-bottom: 60px;">Autoslug is an organization affiliated with the Baskin School of Engineering that specializes in implementing AI, machine learning, and computer vision in practical applications. Our goal is to provide students with opportunities to gain hands-on experience and move beyond the theoretical aspects of these advanced concepts in order to apply them. This would lead to delving into their intersections with humans, data, and various hardware environments.</div>
+    </div>
+    <div class = "centerTile" style = "background-color:rgb(72, 9, 116)">
+        <div style = "font-family: 'Orbitron', sans-serif;font-size:20px; color:white; padding-top: 40px; padding-bottom: 70px; text-align: center; font-size: 50px;">ABOUT US</div>
+        <div class = "midbox" style = "padding-top: 20px; padding-bottom: 20px;">
+            <img src="_assets/groupphoto.png" width = "700px" height = "400px" style = "padding-left: 30px;">
+            <div font-family: style = "font-family: 'Orbitron', sans-serif;font-size:20px; color:white; padding-top: 80px; text-align: center; padding-left: 20px; padding-right: 20px; padding-bottom: 60px;">Autoslug is a Baskin-affiliated organization dedicated to developing practical applications of artificial intelligence (AI), machine learning (ML), and computer vision (CV). We believe that these technologies have the potential to revolutionize many aspects of our lives, and we are committed to providing students with the opportunity to learn how to use them in actual settings. We believe that students should have the opportunity to learn how to use these tools in an applicable real-world environment; to ultimately expand beyond the theoretical side of these advanced concepts and explore their intersection with humans, data, and other forms of hardware. Click <a style = "color:white"href = about.html>here</a> to learn more</div>    
```

Comment:
this is kind of repetitive with the previous paragraph, and with the about page. can we maybe remove this one?